features/issue.feature

```diff
@@ -31,6 +31,116 @@ Feature: hub issue
       """
     And the exit status should be 0
 
+  Scenario: Fetch issues not assigned to any milestone
+    Given the GitHub API server:
+    """
+    get('/repos/github/hub/issues') {
+      assert :milestone => "none"
+
+      json [
+        { :number => 102,
+          :title => "First issue",
+          :state => "open",
+          :user => { :login => "octocat" },
+        },
+      ]
+    }
+    """
+    When I run `hub issue -M none`
```

Comment:
You can change this step to ``When I successfully run `hub issue -M none` `` and you can not assert any output at all. We are just testing that the API endpoint received the correct filtering parameters. If it didn't, the fake API endpoint will return HTTP 400, and the test will fail. We don't need to actually return any records or assert any output here, or in any similar tests.